Code/immersive_launcher/Launcher.cpp

```diff
@@ -47,6 +47,21 @@ LaunchContext* GetLaunchContext()
         return false;                                                                                                  \
     }
 
+
+void SetMaxstdio()
+{
+    const auto handle = GetModuleHandleW(L"API-MS-WIN-CRT-STDIO-L1-1-0.DLL");
```

Comment:
maybe Load instead?

Reply:
as in LoadMaxstdio()?

Reply:
LoadModule lol

Reply:
oh lol, well, yeah i guess, but the dll is already in memory, so i figured GetModuleHandle would be more efficient

Reply:
it's loaded on your PC, might not be on other machines

Reply:
You are referencing a shim dll of ucrtbase.dll, could use that directly.